src/DssSpell.sol

```diff
@@ -106,226 +96,296 @@ contract DssSpellAction is DssAction {
+    address internal constant RWA015_A_OPERATOR          = 0x23a10f09Fac6CCDbfb6d9f0215C795F9591D7476;
     // Custody address
-    address internal constant RWA014_A_COINBASE_CUSTODY      = 0x2E5F1f08EBC01d6136c95a40e19D4c64C0be772c;
-    // -- RWA014 END --
-
-    // -- Spark GNO Onboarding components --
-    address internal constant SPARK_POOL_CONFIGURATOR        = 0xe0C7ec61cC47e7c02b9B24F03f75C7BC406CCA98;
-    address internal constant SPARK_AAVE_ORACLE              = 0x5Cd822d9a4421be687930498ec4B498EB972ad29;
-    address internal constant SPARK_ATOKEN_IMPL              = 0x35542cbc5730d5e39CF79dDBd8976ac984ca109b;
-    address internal constant SPARK_STABLE_DEBT_TOKEN_IMPL   = 0x571501be53711c372cE69De51865dD34B87698D5;
-    address internal constant SPARK_VARIABLE_DEBT_TOKEN_IMPL = 0xb9E6DBFa4De19CCed908BcbFe1d015190678AB5f;
-    address internal constant SPARK_INTEREST_RATE_STRATEGY   = 0xE7Fe5041ec55c229fb41fD9183E5bc24B5E34959;
-    address internal constant SPARK_TREASURY                 = 0x0D56700c90a690D8795D6C148aCD94b12932f4E3;
-    address internal constant SPARK_GNO_ORACLE               = 0xF7CBEF92ef380502dCe9DE8A654A5c1d2A513E78;
-    address internal constant GNO_MEDIANIZER                 = 0x0cd01b018C355a60B2Cc68A1e3d53853f05A7280;
-
-    address internal immutable REGISTRY                      = DssExecLib.reg();
-    address internal immutable MIP21_LIQUIDATION_ORACLE      = DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
-    address internal immutable ESM                           = DssExecLib.getChangelogAddress("MCD_ESM");
-    address internal immutable MCD_VAT                       = DssExecLib.vat();
-    address internal immutable MCD_JUG                       = DssExecLib.jug();
-    address internal immutable MCD_SPOT                      = DssExecLib.spotter();
-    address internal immutable MCD_DAI                       = DssExecLib.dai();
-
-    function onboardRWA014() internal {
-        bytes32 ilk      = "RWA014-A";
+    address internal constant RWA015_A_CUSTODY           = 0x65729807485F6f7695AF863d97D62140B7d69d83;
+
+    // Ilk registry params
+    uint256 internal constant RWA015_REG_CLASS_RWA = 3;
+
+    // RWA Oracle Params
+    uint256 internal constant RWA015_A_INITIAL_PRICE = 2_500_000;
+    string  internal constant RWA015_DOC             = "QmdbPyQLDdGQhKGXBgod7TbQmrUJ7tiN9aX1zSL7bmtkTN";
+    uint48  internal constant RWA015_A_TAU           = 0;
+
+    // Remaining params
+    uint256 internal constant RWA015_A_LINE = 2_500_000;
+    uint256 internal constant RWA015_A_MAT  = 100_00;
+    // -- RWA015 END --
+
+    address internal immutable reg  = DssExecLib.reg();
+    address internal immutable jug  = DssExecLib.jug();
+    address internal immutable spot = DssExecLib.spotter();
+    address internal immutable esm  = DssExecLib.esm();
+    VatLike internal immutable vat = VatLike(DssExecLib.getChangelogAddress("MCD_VAT"));
```

Comment:
We can get rid of the VatLike interface entirely.

It is only used for `ilks` for the PSM offboarding stage 1, which if we aren't reducing global line, we don't need to know the line for the USDP PSM (we can use setIlkDebtCeiling for example).

Vat is still used but not with the VatLike interface in this spell :)

In the future, please use [` DssExecLib.vat();`](https://github.com/makerdao/dss-exec-lib/blob/master/src/DssExecLib.sol#L170) please and adapt code to fit the pattern of declaring addresses on top line and using 'Like' interfaces inline :)